### ISSUE TITLE
Custom template for replies

### ISSUE THREAD

**Issue by notramo** [matrix-org/matrix-appservice-irc#1016]
Some bridge admins would prefer a custom format for reply messages. https://github.com/elves/elvish/issues/871#issuecomment-601636663
It should be a simple template:
`message`: **Yes, I agree.**
`nick` (sender of the original message): **example_account**
`original` (replied message): **We should push the release to the repo.**
```yaml
replyTemplate: "{{message}} (re: {{nick}}: {{original}}"
```
> Yes, I agree. (re: example_account: We should push the release to the repo.)

```yaml
replyTemplate: "<{{nick}}: {{original}}> {{message}}"
```
> <example_account: We should push the release to the repo.> Yes, I agree.

Without quoting (some IRC channel admins would prefer not to quote the original message at all):
```yaml
replyTemplate: "{{nick}}: {{message}}"
```
> example_account: Yes, I agree.

**Comment by tadzik** [comment]
Implemented in #1424

**Comment by tadzik** [comment]
Fixed now, and should land in the next release :)
